recipes/prophet/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "prophet" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3e682e8ea6e1ee26a92cf289f207d539f30e44879126c128ff8f4e360eb25a8b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  binary_has_prefix_files:
+    - Lib/site-packages/prophet/stan_model/prophet_model.pkl  # [win]
+    - {{ SP_DIR }}/stan_model/prophet_model.pkl  # [not win]
```

This should fix the issue on Windows

```suggestion
    - {{ SP_DIR|replace("\\", "/") }}/stan_model/prophet_model.pkl
```

---

Cool, it did!